src/main/java/com/aws/greengrass/secretmanager/SecretManagerService.java

```diff
@@ -56,6 +58,14 @@
     private final AuthorizationHandler authorizationHandler;
     private final ExecutorService executor;
     private final AtomicReference<Future<?>> syncFuture = new AtomicReference<Future<?>>(null);
+    private final ChildChanged handleConfigurationChangeLambda = (whatHappened, node) -> {
+        if (whatHappened == WhatHappened.timestampUpdated || whatHappened == WhatHappened.interiorAdded) {
```

Comment:
Why skip interior added?

Reply:
That event does not require an update in config.